gcp_deepvariant_runner.py

```diff
@@ -871,16 +889,43 @@ def run(argv=None):
       help=('Optional space-separated list of regions to process. Elements can '
             'be region literals (chr20:10-20) or Google Cloud Storage paths '
             'to BED/BEDPE files.'))
+  parser.add_argument(
+      '--preemptible',
+      default=False,
+      action='store_true',
+      help=('Use preemptible VMs for the pipeline and all re-tries in case '
+            '--attempts is set to larger than 1.'))
+  parser.add_argument(
+      '--attempts',
+      type=int,
+      default=0,
+      help=('Maximum number of times to attempt running each worker (within a '
+            'job). Depending on --preemptible flag re-attempts will be using '
+            'regular (non-preemptible) VMs or preemptible VMs. Note regular '
+            'VMs may still crash unexpectedly, so it may be worth to retry on '
+            'transient failures.'))
+  # TODO(#22): deprecate this flag in next release in favour of --attempts.
   parser.add_argument(
       '--max_non_preemptible_tries',
       type=int,
       default=2,
-      help=('Maximum number of times to try running each worker (within a job) '
+      help=('WARNING: This flag will be deprecated soon. '
```

Comment:
"WARNING: This flag is deprecated and will be removed in a future release.  Use --attempts and --preemptible to control retry behaviour"

and drop the rest of the documentation for the flag

Reply:
Done.